### ISSUE TITLE
get_suggested_follows_by_actor fails with ValidationError for Response

### ISSUE THREAD

**Issue by temujin9** [MarshalX/atproto#205]
The error suggests to me that the underlying semantic changed out from under this codebase.

```
Traceback (most recent call last):
  File "/home/temujin9/.local/lib/python3.10/site-packages/atproto/xrpc_client/models/utils.py", line 87, in _get_or_create
    return model(**model_data)
  File "/home/temujin9/.local/lib/python3.10/site-packages/pydantic/main.py", line 164, in __init__
    __pydantic_self__.__pydantic_validator__.validate_python(data, self_instance=__pydantic_self__)
pydantic_core._pydantic_core.ValidationError: 8 validation errors for Response
suggestions.0.banner
  Extra inputs are not permitted [type=extra_forbidden, input_value='https://cdn.bsky.app/img...u6kla7qb4fhafdihti@jpeg', input_type=str]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.0.followsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=84, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.0.followersCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=50, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.0.postsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=593, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.banner
  Extra inputs are not permitted [type=extra_forbidden, input_value='https://cdn.bsky.app/img...7dftfyvh6j46diolje@jpeg', input_type=str]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.followsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=279, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.followersCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=381, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.postsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=2529, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/temujin9/Projects/T9_Productions/code/bsky_scripts/suggested_follows", line 24, in <module>
    result = graph_ns.get_suggested_follows_by_actor( params )
  File "/home/temujin9/.local/lib/python3.10/site-packages/atproto/xrpc_client/namespaces/sync_ns.py", line 817, in get_suggested_follows_by_actor
    return get_response_model(response, models.AppBskyGraphGetSuggestedFollowsByActor.Response)
  File "/home/temujin9/.local/lib/python3.10/site-packages/atproto/xrpc_client/models/utils.py", line 98, in get_response_model
    return get_or_create(response.content, model)
  File "/home/temujin9/.local/lib/python3.10/site-packages/atproto/xrpc_client/models/utils.py", line 64, in get_or_create
    raise e
  File "/home/temujin9/.local/lib/python3.10/site-packages/atproto/xrpc_client/models/utils.py", line 57, in get_or_create
    model_instance = _get_or_create(model_data, model, strict=strict)
  File "/home/temujin9/.local/lib/python3.10/site-packages/atproto/xrpc_client/models/utils.py", line 89, in _get_or_create
    raise ModelError(str(e)) from e
atproto.exceptions.ModelError: 8 validation errors for Response
suggestions.0.banner
  Extra inputs are not permitted [type=extra_forbidden, input_value='https://cdn.bsky.app/img...u6kla7qb4fhafdihti@jpeg', input_type=str]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.0.followsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=84, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.0.followersCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=50, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.0.postsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=593, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.banner
  Extra inputs are not permitted [type=extra_forbidden, input_value='https://cdn.bsky.app/img...7dftfyvh6j46diolje@jpeg', input_type=str]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.followsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=279, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.followersCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=381, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
suggestions.1.postsCount
  Extra inputs are not permitted [type=extra_forbidden, input_value=2529, input_type=int]
    For further information visit https://errors.pydantic.dev/2.5/v/extra_forbidden
```

Usage that lead to this was unremarkable:
```
  params = graph_m.get_suggested_follows_by_actor.Params(actor=profile.did)
  result = graph_ns.get_suggested_follows_by_actor( params )
```

**Comment by MarshalX** [comment]
Sdk version?

**Comment by MarshalX** [comment]
particularly fixed in v0.0.34. pls update. still waiting for the fix from bsky team
